src/go/rpk/pkg/config/config.go

```diff
@@ -85,6 +86,11 @@ func defaultMap() map[string]interface{} {
 		"port":    9092,
 	}
 	var defaultListeners []interface{} = []interface{}{defaultListener}
+	var defaultAdminListener interface{} = map[string]interface{}{
+		"address": "0.0.0.0",
```

Comment:
I think all listeners now should have a name, right? 🤔

Reply:
I see that we don't do it for kafkaapi but I kind of feel like we should have it even there, right? (not implying that should change in this PR)

Reply:
#1137